src/mscorlib/shared/System/Diagnostics/Tracing/TraceLogging/EventSourceActivity.cs

```diff
@@ -77,8 +77,8 @@ public Guid RelatedId
 
         /// <summary>
         /// Writes a Start event with the specified name and data.   If the start event is not active (because the provider 
-        /// is not on or keyword-level indiates the event is off, then the returned activity is simply the 'this' poitner 
-        /// and it is effectively like the Start d
+        /// is not on or keyword-level indicates the event is off, then the returned activity is simply the 'this' pointer 
+        /// and it is effectively like the Start d           ^^ indicates? also sentence seems incomplete?
         /// 
```

Comment:
And this one is hard to decode. The last sentence also seems incomplete here.

Reply:
Yes, *indicates*. The reference source version has the same incomplete sentence. If nobody has a better idea, I'd suggest just changing the sentence to end after *'this'.*

Reply:
@brianrob can clarify what he meant

Reply:
@brianrob  ?

Reply:
Indicates is correct.  The sentence is definitely cut off.  Let's go with "then the returned activity is simply the 'this' pointer and it is effectively like start did not get called."

Reply:
Cool. Thanks @brianrob!

Also @danmosemsft , should I rebase this PR? Seems like we got a few more merge conflicts to resolve since 81a78ef (2 days ago).

Reply:
@bchavez yes please rebase